### ISSUE TITLE
[BUG] [Feature] [HOTFIX] Improve Performance

### ISSUE THREAD

**Issue by andreespirela** [mandarineorg/mandarinets#87]
**[Details]**
Right now, Mandarine is presenting the following performance. 
```
Running 10s test @ http://localhost:1447
  10 threads and 40 connections
  Thread Stats   Avg      Stdev     Max   +/- Stdev
    Latency    10.39ms    3.49ms 116.70ms   94.86%
    Req/Sec   392.12     50.56   760.00     89.80%
  39170 requests in 10.06s, 9.97MB read
Requests/sec:   3894.13
Transfer/sec:      0.99MB
```

As you can see. That is extremely bad.

**[Expected behavior]**
It should be above 10k per second.

**[OS]**
MacOS Mojave 10.14.6

**[Environment]**
 - Mandarine version: 1.2.1
- deno 1.2.0
- v8 8.5.216
- typescript 3.9.2

**[Additional]**
Thanks the Drashland team for getting the performance for Mandarine.
CC: @crookse , @ebebbington .

**Comment by andreespirela** [comment]
We have found the main cause of this. It is mostly related to the way we handle the session container & the CORS headers.

**Comment by andreespirela** [comment]
As further investigation was released. We have discovered that the command used for the benchmark which is `wrk -c40 -d10 -t10 [URL]` was not right. This is because, Mandarine creates a new session object inside the session container every time a new request with a new context is received. Mandarine handles the creation of a new session object inside the session container **for the initial connection**, this means, after you have made your first connection, Mandarine will create a cookie inside your browser context and so Mandarine does not have to deal with creating a new session object anymore rather than retrieving the current one. There are ways to trick Mandarine's session container (which is the way it's meant to be) such as using an _Incognito Windows_, in an incognito windows, your browser creates a new connection context which means, every incognito window you have open will result in the creation of a new session cookie thus a new session object inside the session container. WIth that said, what `wrk -c40 -d10 -t10` is doing, is creating multiple **new browser context** for each connection, forcing Mandarine to create a new session object for every connection it receives from the command `wrk`, this results in a performance impact since `wrk` requests about 35k every 10 seconds which is the equivalent of 35 session objects every 10 seconds, in any case, this is not a real production issue. If we remove the session container and we use `wrk` we can see how Mandarine is very well capable of handling 10k requests per second which means, 100k every 10 seconds, this is just the result of not having to create multiple session objects inside the session container. Esentially, `wrk` behaves as an incognito window for every request it makes. 

Closing as benchmark is miscalculated.
